test/include/dlaf_test/matrix/util_generic_blas.h

```diff
@@ -17,29 +17,39 @@
 
 #include "dlaf_test/util_types.h"
 
+/// @file
+
 namespace dlaf {
 namespace matrix {
 namespace test {
 
 using namespace dlaf_test;
 
-/// @brief Returns el_op_a, el_b, res_b for side = Left.
+/// Returns elements of the three matrices, solving the triangular matrix equation
+/// op(A) B = X (with A on side == Left).
+///
+/// The tile elements of A matrix (el_op_a) are chosen such that:
+///   op(A)_ik = (i+1) / (k+.5) * exp(I*(2*i-k)) for the referenced elements
+///   op(A)_ik = -9.9 otherwise.
+///
+/// The elements of B (el_b) should be:
+/// B_ij = (Sum_k op(A)_ik * res_Xkj) / alpha
+///      = (op(A)_ii * res_Xij + (kk-1) * gamma) / alpha,
+/// where gamma = (i+1) / (j+2) * exp(I*(2*i+j)),
+///       kk = i+1 if op(a) is an lower triangular matrix, or
+///       kk = m-i if op(a) is an lower triangular matrix.
+/// Therefore
+/// B_ij = (res_Xij + (kk-1) * gamma) / alpha, if diag == Unit
+/// B_ij = kk * gamma / alpha, otherwise.
+///
+/// Finally, the X matrix elements (res_b) are computed as
+///   res_Xkj = (k+.5) / (j+2) * exp(I*(k+j)),
```

Comment:
Res is not defined above. just use X.
```suggestion
///   X_kj = (k+.5) / (j+2) * exp(I*(k+j)),
```
I would also rename the object `res_b` to `el_x`, to be more consistent with the doc.